src/api/.haml-lint.yml

```diff
@@ -59,3 +59,4 @@ exclude:
 - 'app/views/statistics/**/*'
 - 'app/views/status/**/*'
 - 'app/views/webui/**/*'
+- 'app/views/webui2/webui/kaminari/**/*'
```

Comment:
What is the reason for not linting the Kaminari views?

Reply:
The kaminari views are _generated_ and I don't see the point to change them to make them pass our lint tests.

Reply:
Alright, it makes sense. Could we have a comment for this?

Reply:
Instead of adding a comment to every view, I've commented it in the commit description.

Reply:
My idea is to add a comment in this file, just above the new line.

Reply:
:ok: